workgraph/android.yml

```diff
@@ -117,12 +122,25 @@ android-nightlies-manual-test:
     dependencies:
         - android-nightlies-tier2
 
+android-x86-nightlies-manual-test:
+    # this is update testing, possibly done by triggering some of QA's tests against a nightly made on date
```

Comment:
You can put these sorts of things in the `description:` so they're visible on the site